pom.xml

```diff
@@ -93,6 +93,21 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.heigit.ohsome</groupId>
+      <artifactId>oshdb-api-ignite</artifactId>
+      <version>${oshdb.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.slf4j</groupId>
+          <artifactId>slf4j-simple</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.slf4j</groupId>
+          <artifactId>slf4j-log4j12</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
```

Comment:
This is write, but what do you think to remove the dependency of 
- sl4j-simple
from the oshdb-api. So that we don't need to exclude this in such an situation.
We should create a new issue for that.

Reply:
>  We should create a new issue for that.

:+1: yeah, this would be an issue for the oshdb repo.